mmv1/third_party/terraform/services/compute/resource_compute_router_peer.go.erb

```diff
@@ -881,6 +944,22 @@ func flattenNestedComputeRouterBgpPeerManagementType(v interface{}, d *schema.Re
 	return v
 }
 
+func flattenNestedComputeRouterBgpPeerMd5AuthenticationKey(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+	if v == nil {
+		return nil
+	}
+	originalKeyValue := d.Get("md5_authentication_key").([]interface{})
+	transformed := make(map[string]interface{})
+	transformed["name"] = v
+	log.Printf("[DEBUG] originalKeyValue - %#v)", originalKeyValue)
+	//key value is not returned as it is a sensitive field
+	if len(originalKeyValue) != 0 {
+		transformed["key"] = originalKeyValue[0].(map[string]interface{})["key"]
+		log.Printf("[DEBUG] transformed key value - %#v)", transformed["key"])
```

Comment:
This will log the key into debug logs, which may not be what we want if it's a sensitive value. It will always be in the HTTP request logs, but it may make sense to remove this